### DISCUSSION TITLE
invalid group characteristic

### DISCUSSION THREAD

**Discussion by EdwardWong1994** [neovim/neovim#24940] [General]
### Problem

hello, everytime i open the nvim, it will occur `invalid group characteristic`.  At first i bear with it but the more i use `vim`, the more pain it give me.  Now i have a lot of configuration files and have no idea what can be the problems be?

Does anyone help?

### Steps to reproduce

<img width="767" alt="image" src="https://github.com/neovim/neovim/assets/96438179/b4738337-7a54-40ef-9694-f02d84ca0099">

I have attached my configuration file here.  I would appreciate who can help me this out.  

### Expected behavior

suspected it do not give me such warning error

### Neovim version (nvim -v)

0.9.1

### Vim (not Nvim) behaves the same?

did not use vim anymore

### Operating system/version

macOS11.4

### Terminal name/version

iterm

### $TERM environment variable

xterm-256color

### Installation

from source

**Comment by EdwardWong1994** [comment]
Sorry, forget to attach my link [nvim configuration file](https://github.com/EdwardWong1994/dotfile.git)

**Comment by zeertzjq** [comment]
Try updating you plugins.

**Reply by EdwardWong1994** [reply]
no use to update the plugins.
